reconcile/dynatrace_token_provider/integration_v2.py

```diff
@@ -0,0 +1,20 @@
+from reconcile.dynatrace_token_provider.meta import QONTRACT_INTEGRATION
+from reconcile.utils.runtime.integration import (
+    PydanticRunParams,
+    QontractReconcileIntegration,
+)
+
+
+class DynatraceTokenProviderIntegrationParamsV2(PydanticRunParams):
+    ocm_organization_ids: set[str] | None = None
+
+
+class DynatraceTokenProviderIntegrationV2(
+    QontractReconcileIntegration[DynatraceTokenProviderIntegrationParamsV2]
+):
+    @property
+    def name(self) -> str:
+        return QONTRACT_INTEGRATION
+
+    def run(self, dry_run: bool) -> None:
+        pass
```

Comment:
Just to be clear, this is intentionally left empty, right?

Reply:
Good question. Yeah its intentional. 
Im essentially trying to have a clean workspace without any schema changes.
Let me adjust the PR description.